docs/tutorials/scenarios_suites/MetaShift.rst

```diff
@@ -0,0 +1,72 @@
+MetaShift
+-----------------
+
+MetaShift can be described as a dataset of dataset. It divides the original Visual Genome dataset into classes and contexts in order to capture distribution shifts.
+See paper : https://arxiv.org/abs/2202.06523
+
+The MetaShift class is built with ContinualScenario in mind. But it also lets the user select specific classes or training ids. 
+As many images can be found in more than one class(context) configuration, the user can specify that all images should aprear in exactly one configuration.
```

Comment:
peut etre qu'une autre facon de formuler ca serait:

In continuum, we use the context information to build various types of scenarios, notably domain incremental scenario.
The user can select a subset of classes to include in the scenario with `class_names` argument and if all classes should appear in all tasks with `???` arguments.
Images can be associated with several context, then the user can decide if one image appear only once or can appear several time.
If the image should appear only once, then the context chosen will be selected randomly.